packages/runtime/src/runtime/navigateApi.tsx

```diff
@@ -0,0 +1,65 @@
+import type {
+  NavigateScreenAction,
+  CustomScope,
+  ScreenContextDefinition,
+  EnsembleStorage,
+} from "@ensembleui/react-framework";
+import { findExpressions, evaluate } from "@ensembleui/react-framework";
+import { isString, cloneDeep, set } from "lodash-es";
+import type { NavigateFunction, Location } from "react-router-dom";
+
+export interface BrowserHistoryDefinition {
+  navigate: NavigateFunction | undefined;
+  location: Location | undefined;
+}
+
+export const browserHistory: BrowserHistoryDefinition = {
```

Comment:
Don't use a global object

Reply:
I had idea about the using atom, but then I realized that the jotai is not used in the runtime repo. So I used like this.

Reply:
And I think this will not cause any issue and useful in other APIs as well, where we need the access of the useNavigate.